contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.3.0--2.4.0.sql

```diff
@@ -81,6 +81,62 @@ GRANT SELECT ON information_schema_tsql.schemata TO PUBLIC;
 
 -- please add your SQL here
 
+-- DATABASE_PRINCIPALS: Include hard coded public, sys and INFORMATION_SCHEMA
```

Comment:
Let's only target it to PG15 unless anyone asks

Reply:
removed the code change in ``babelfishpg_tsql--2.3.0--2.4.0.sql`